.gitignore

```diff
@@ -1,6 +1,7 @@
 __pycache__/
 .idea
-bandit_scan_results.sarif
 scripts/inputs
 scripts/outputs
 scripts/*.bat
+=======
```

Comment:
Was '=======' accidentally included? Possibly from a merge conflict?

Reply:
Yes that was accidental. Updated